ddtrace/contrib/langchain/patch.py

```diff
@@ -154,7 +176,11 @@ def traced_llm_generate(langchain, pin, func, instance, args, kwargs):
                 span.set_tag_str("langchain.request.%s.parameters.%s" % (llm_provider, param), str(val))
 
         completions = func(*args, **kwargs)
-        if isinstance(instance, langchain.llms.OpenAI):
+        if (
+            isinstance(instance, BASE_LANGCHAIN_MODULE.llms.OpenAI)
+            or langchain_openai
+            and isinstance(instance, langchain_openai.OpenAI)
```

Comment:
Question `if True or False and True` will evaluate to `True` and . Is this the expected behavior? Do we need brackets here?
```suggestion
            isinstance(instance, BASE_LANGCHAIN_MODULE.llms.OpenAI)
            or (langchain_openai
            and isinstance(instance, langchain_openai.OpenAI))
```

Reply:
Great catch!